financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/ManifestExtensions.kt

```diff
@@ -25,6 +25,12 @@ internal fun FinancialConnectionsSessionManifest.getRedactedEmail(): String? =
         }
     }
 
+internal fun FinancialConnectionsSessionManifest.shouldDisableAuthSessionRetrieval(): Boolean =
+    features
+        ?.get("bank_connections_disable_defensive_auth_session_retrieval_on_complete")
+        // if feature not present, disable auth session retrieval
```

Comment:
important to get right here, so if this flag is set, we should disable and return true, right? or do we have a double negative here?

Reply:
inverted the flag for easier readability + added some tests.

It'd retrieve the session *unless* the `disable` killswitch is set to true. https://github.com/stripe/stripe-android/pull/7125/commits/2e5d52d593f54f36cd785c89118f18bc27c6d6a7**